rediscluster/exceptions.py

```diff
@@ -74,3 +74,8 @@ class MovedError(AskError):
     """
     """
     pass
+
+class MasterDownError(ClusterDownError):
```

Comment:
New exception class but unused right now. It is imported in previous file but not used there.

Reply:
I've added it to the exception parser in connection.py. It's a subclass of ClusterDownError, so it'll just be treated the same throughout the client, since the cause of both errors are pretty much the same.